chain/consensus/iface.go

```diff
@@ -71,6 +72,11 @@ func ValidateBlockPubsub(ctx context.Context, cns Consensus, self bool, msg *pub
 		return pubsub.ValidationReject, what
 	}
 
+	if !cns.IsEpochInConsensusRange(blk.Header.Height) {
+		log.Warnf("received block outside of consensus range (%d)", blk.Header.Height)
+		return pubsub.ValidationReject, "invalid_block_height"
```

Comment:
maybe ValidationIgnore?

Can this happen by accident or normal operation? If thats the case, it should br ValidationIgnore.